classes/template/PKPTemplateManager.php

```diff
@@ -815,7 +816,17 @@ public function requiresVueRuntime() {
                     'contexts' => ['frontend']
                 ]
             );
-    
+
+            $request = Application::get()->getRequest();
+            // Include css styles used by UI-Library Vue.js components
+            $this->addStyleSheet(
+                'build',
+                $request->getBaseUrl() . '/styles/build.css',
+                [
+                    'priority' => self::STYLE_SEQUENCE_LAST,
+                    'contexts' => 'frontend',
+                ]
+            );
```

Comment:
This is a workaround, and might need to be changed. I added it so that the styles for the ui-library components are available when using said components in the front-office